test-fixtures/fix/fix-general/05/metadata.json

```diff
@@ -1,4 +1,6 @@
 {
     "description": "Should succeed in adding a BTJ SID with BIB_FORMAT_SETTINGS",
-    "settings": "BIB_FORMAT_SETTINGS"
+    "settings": "BIB_FORMAT_SETTINGS",
+    "only": true,
```

Comment:
All tests should be run? So all only-flags should be false and if skips are necessary skips true?